quilter.py

```diff
@@ -1,3 +1,14 @@
+###############################################################################
+#
+# Project:  Quilter utility for raster.utah.gov
+# Purpose:  Download, merge, and reproject multiple datasets from the same
+#           product into a single file
+# Author:   Jacob Adams, jdadams@utah.gov
+#
+###############################################################################
+# Copyright (c) 2019 AGRC
```

Comment:
There is [much](https://www.copyright.gov/circs/circ03.pdf) [discussion] (https://www.copyrightlaws.com/copyright-symbol-notice-year/) [on this] (https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated). My preference is to use the date first published.

Reply:
I'd rather you not put any copyright text inside the document and only use the license file in the repository. Please remove it.

Reply:
Thinking about that whole header, that's something I've been using for a while based on the GDAL code. Should I move all that info (minus the copyright text) into the docstring and delete the header?

Reply:
I would add the agrc license and the GDAL license to the repository if required. The header is not necessary in my opinion other than describing the purpose of the file. Everything is implied from the repository, the licenses, and purposes described in the readme. It's mainly a duplication.

Reply:
👍 

GDAL uses the MIT license, which is the same as this project (and seems to be the preferred license for open AGRC projects?). I'm going to leave it in ogrmerge.py as a courtesy because it was written by them, but I'll get rid of it on quilter.